sql/core/src/test/resources/tpcds-query-results/v1_4/q23b.sql.out

```diff
@@ -3,7 +3,4 @@
 -- !query schema
 struct<c_last_name:string,c_first_name:string,sales:decimal(28,2)>
 -- !query output
-NULL	Robert              	598.86
-Brown                         	Monika              	6031.52
-Collins                       	Gordon              	727.57
-Green                         	Jesse               	9672.96
```

Comment:
I've checked that the result is the same when using StringType instead of CharType. It looks sf=1 is not enough to output valid rows, e.g., if sf=20, this result will be;
```
+NULL   NULL    1211190.10
+NULL   NULL    2162596.43
+NULL   Amber                   26523.81
+NULL   Amelia                  39837.88
+NULL   Angela                  19000.56
+NULL   Ann                     9771.24
+NULL   Anthony                 33085.99
+NULL   Antonia                 9686.34
+NULL   Ashley                  44888.09
+NULL   Beatrice                42874.69
+NULL   Bernadette              33897.93
+NULL   Brandon                 36527.85
+NULL   Brian                   14629.00
+NULL   Bryan                   8213.72
+NULL   Bryce                   37041.48
+NULL   Camille                 3828.89
+NULL   Carl                    39187.08
+NULL   Chang                   13063.80
+NULL   Charity                 54843.39
+NULL   Charlene                49054.79
+NULL   Charles                 4771.75
+NULL   Charles                 62057.20
+NULL   Charlotte               16385.30
+NULL   Cherie                  27987.22
+NULL   Christopher             36433.14
+NULL   Clarence                25626.84
+NULL   Corey                   23496.48
+NULL   Dale                    5110.33
+NULL   Daniel                  22358.95
+NULL   Darlene                 57923.08
+NULL   Derek                   49167.25
+NULL   Dexter                  10170.43
+NULL   Donald                  24886.59
+NULL   Donna                   52864.24
+NULL   Edith                   61615.29
+NULL   Elizabeth               23627.73
+NULL   Elmer                   19619.60
+NULL   Eloise                  18450.24
+NULL   Erick                   38025.12
+NULL   Erika                   27798.27
+NULL   Evan                    46451.24
+NULL   Fay                     29404.94
+NULL   Frank                   38593.81
+NULL   Gary                    26419.34
+NULL   Giuseppina              23983.27
+NULL   Grace                   10196.36
+NULL   Gretchen                14514.03
+NULL   Guadalupe               25678.72
+NULL   Helena                  25266.03
+NULL   Howard                  19108.80
+NULL   Irene                   21181.11
+NULL   Iris                    20347.29
+NULL   Isaac                   21409.78
+NULL   Isabel                  10981.56
+NULL   James                   20208.41
+NULL   James                   45018.22
+NULL   Janet                   23444.60
+NULL   Janice                  11866.23
+NULL   Janice                  23576.67
+NULL   Jason                   3513.97
+NULL   Jason                   13569.16
+NULL   Jayne                   32662.64
+NULL   Jeffery                 12079.50
+NULL   Jeffrey                 23556.44
+NULL   Jennifer                33641.35
+NULL   Jessica                 21085.21
+NULL   John                    27934.22
+NULL   Jose                    27518.94
+NULL   Joseph                  105492.82
+NULL   Josephine               31305.22
+NULL   Julio                   33649.91
+NULL   Kaleigh                 26772.97
+NULL   Karoline                17438.02
+NULL   Keith                   11594.46
+NULL   Kenneth                 8982.40
+NULL   Kenneth                 37143.32
+NULL   Kerri                   14295.48
+NULL   Kimberly                37517.97
+NULL   Lara                    7898.06
+NULL   Laura                   10704.81
+NULL   Lillian                 27638.75
+NULL   Lina                    30780.78
+NULL   Linda                   21437.87
+NULL   Lori                    40834.58
+NULL   Lucas                   34701.86
+NULL   Lucille                 31975.22
+NULL   Luis                    64716.14
+NULL   Lynette                 39160.74
+NULL   Marcus                  65630.87
+NULL   Marjorie                32153.42
+NULL   Marlene                 28037.14
+NULL   Melanie                 46728.54
+NULL   Michael                 37021.70
+NULL   Michael                 44109.43
+NULL   Naomi                   15056.09
+NULL   Nicholas                30684.17
+NULL   Norman                  36903.91
+NULL   Oscar                   96584.20
+NULL   Pamela                  39836.80
+NULL   Paul                    18862.79
```